api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onEditorPreparing.md

```diff
@@ -393,9 +255,13 @@ Use the **parentType** function parameter to check if the editor being customize
 
 - Implement other customization cases.
 
+Use the [onInitNewRow]({basewidgetpath}/Configuration/#onInitNewRow) handler to set default cell values for newly created rows.
```

Comment:
This sentence makes me wonder why it is given in an article about onEditorPreparing. We must link it to onEditorPreparing somehow. For example,

"onEditorPreparing in not an appropriate function to set a default value for an editor. Use the onInitNewRow function for this purpose."

And maybe give it as a note after the demo button.